src/com/carolinarollergirls/scoreboard/utils/ScoreBoardClock.java

```diff
@@ -0,0 +1,66 @@
+package com.carolinarollergirls.scoreboard.utils;
+
+import com.carolinarollergirls.scoreboard.defaults.DefaultScoreBoardEventProvider;
```

Comment:
I don't think we should have a circular dependency between packages like this, this class should go in defaults.